examples/presence_state.rs

```diff
@@ -1,16 +1,24 @@
+use std::collections::HashMap;
+
 use pubnub::{Keyset, PubNubClientBuilder};
-use serde::Serialize;
```

Comment:
Linter didn't liked it oO

Reply:
Oooh, maybe they changed some of politics. 
Or maybe it was because State2 was not used during the check (just assuming - I don't know why it disagreed with that ;o)